### DISCUSSION TITLE
2023-3-25 软件开发技能与机动车驾驶技能 ~ 蒋继发 (ThaddeusJiang)

### DISCUSSION THREAD

**Discussion by giscus** [ThaddeusJiang/thaddeusjiang.github.io#37] [Announcements]
# 2023-3-25 软件开发技能与机动车驾驶技能 ~ 蒋继发 (ThaddeusJiang)

开发工具越来越强大了，AI 可以辅助编程了，但是我们真的在工具的加持下提高了自己的编程技能了吗？生成式AI（Generative AI）大行其道的时代背景下，我们使用 AI 辅助编程的方式真的是正确的吗？我们真的变得高效了吗？我们是在朝着正确的方向发展吗？

https://thaddeusjiang.com/2023-3-25-ruan-jian-kai-fa-ji-nen-yu-ji-dong-che-jia-shi-ji-nen.html

**Comment by mounan** [comment]
可是机动车也要自动驾驶了

**Reply by ThaddeusJiang** [reply]
是啊。希望机动车能保留一个设置目的地的功能，不要让我一遍又一遍的用自然语言告诉它。

开车去超市：

👤去超市；
🤖是要去 A 超市吗？
👤不是，今天要去 B 超市。
🤖 帮你找到了 3 个 B 超市，要去最近的吗？
👤是的。

😂😂
